source/FunkinLua.hx

```diff
@@ -697,6 +697,10 @@ class FunkinLua {
 			}
 		});
 
+		Lua_helper.add_callback(lua, "getPref", function(pref:String, ?defaultValue:Dynamic) {
+			return ClientPrefs.getPref(pref, defaultValue);
+		});
+
```

Comment:
Is there gonna be setPref?

Reply:
> Is there gonna be setPref?

yeah, my bad--i was in a rush, but i can make that soon